ortools/linear_solver/xpress_interface_test.cc

```diff
@@ -730,30 +730,24 @@ TEST_F(XpressFixtureMIP, Write) {
   tmpFile.close();
   std::filesystem::remove_all(temporary_working_dir);
 
-  EXPECT_EQ(tmpBuffer.str(), R"(NAME          newProb
-OBJSENSE  MAXIMIZE
-ROWS
- N  __OBJ___
- L  R1
- L  R2
-COLUMNS
-    C1        __OBJ___  1
-    C1        R1        3
-    C2        __OBJ___  2
-    C2        R1        1.5
-    C2        R2        -1.1122334455667788
-RHS
-    RHS00001  R1        1
-    RHS00001  R2        5
-RANGES
-    RNG00001  R2        2
-BOUNDS
- UI BND00001  C1        9
- LO BND00001  C1        -1
- UP BND00001  C2        5.147593849384714
- LO BND00001  C2        -1
-ENDATA
-)");
+  std::string expectedMps =
```

Comment:
Why not keeping the Raw string literal `R"` ?
Personally, I feel it more easier to read (thant splitting line by line with all `+ "\n"` at the end), or when you want to diff terminal output against the source file

note you can also use `// clang-format off` / `// clang-format on` to disable/enable formatting on a section of the code (e.g. data matrix manually formatted).

Reply:
we had to change this because we saw that sometimes trailing spaces at the end of lines were automatically removed by IDEs (for example, the `" L  R1              "` line), but they're actually part of the expected MPS. If you prefer, I can reduce this change to the impacted lines.

Reply:
OK keep it like you have done !